lib/hci-socket/hci.js

```diff
@@ -86,7 +87,7 @@ Hci.prototype.init = function() {
 
   var deviceId = process.env.NOBLE_HCI_DEVICE_ID ? parseInt(process.env.NOBLE_HCI_DEVICE_ID, 10) : undefined;
 
-  this._socket.bindRaw(deviceId);
+  this._deviceId = this._socket.bindRaw(deviceId);
```

Comment:
Since this._socket is actually a BluetoothHciSocket object, when I follow the code through to the BluetoothHciSocket class I'm seeing it's bindRaw method return an int BluetoothHciSocket::bindRaw(int* devId) { .. } (I have BluetoothHciSocket 5.0 installed).  Would that not be correct?  When I write out the this._deviceId value  I always get what was defined with NOBLE_HCI_DEVICE_ID parameter which is set through the bindRaw call.